jobs/deploy-znd.groovy

```diff
@@ -327,9 +327,6 @@ def deployCronYaml() {
 // just do it always.
 def deployToGatewayConfig() {
    dir("webapp") {
-       exec(["make", "-C", "services/graphql-gateway",
-             "deploy-gateway-config",
-             "DEPLOY_VERSION=${VERSION}"]);
        if (fileExists("services/queryplanner/Makefile")) {
```

Comment:
doesn't this if also need to disappear (just the if, not the code in the if) similar to #108 ?

Reply:
I wasn't sure, but Craig pointed it out too. It is removed.

Reply:
Oops... something went wrong with my second commit. Huh.

Reply:
NVM, it is fixed now.